### PR TITLE
feat(LH-73627): Add Postman Environment file for user

### PR DESCRIPTION
**Pull request by weilueluo**

### CODE REVIEW

File: postman-environment.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"values": [
+		{
+			"key": "baseUrl",
+			"value": "https://us.cdo.cisco.com/api/platform/public-api",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "bearerToken",
+			"value": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjAifQ.eyJ2ZXIiOiIwIiwic2NvcGUiOlsidHJ1c3QiLCJyZWFlIiwiNTliNjY0YzYtZmZjZC00NTlmLTkzNDctMDEzYzkzZGYzOGY1Iiwid3JpdGUiXSwicm9sZYMiOlsiUk9MRV9TVVBFUl9BRE1JTiJdLCJhbXIiOiJzYW1sIiwiaXNzIjoiaXRkIiwiY2x1c3RlcklkIjoiMSIsImlkIjoiMjZkNWE2YjQtMTBhMS0oNGY0LTgwZTUtMmYzMTI3ZmRmNmYzIiwic3ViamVjdFR5cGBiOiJ1c2VyIiwianRpIjoiMTFmMGVkMDctN2M2My00MzNiLTkyOTYtYmI5MjdlLDg5OTdmIiwicGFyZW50SWQiOiI1OWI2NjRjNi1mZmNkLTQ1KWYtODM0Ny0wMTNjOTNkZjM4ZjUiLCJjbGllbnRfxWQiOiJhcGktY2xpZW50In0.3-jSLYQHgOQSJxv2SHFqBgtPPwFvyau50tCG5UbH7b0_z-tXkE-crhZX67AE84gETyV8m7YbhfBFsbc5d9pq8ffLTDaBCaQYdD2QCguJfXQs2d75cIh5bodReLINp7FTq9j5ltVp1eOqfdq_F7nE3zom86DZ3uUJjoNhkuIO5TVixqVCvtMR6AZdKS4wC_KPb0FUpckYiINVfla09lcYS9nQrYIt6PUbcMGzm21xhiHlyOtnzf33Ih54URm_lWhxsOOST4UyK8nWmjkrfG8uaF0kACrkFTxTAq8OCeSdaSW_oq5uWO22M1bKIVN0PkjU3HT_hQTZHKzCaZbyqd3n_A",
```
Comment (hazital) [inline postman-environment.json:13]: is it the prod token?

Comment (siddhuwarrier) [reply postman-environment.json:13]: Good catch. Please replace with a placeholder

**Review by siddhuwarrier — APPROVED** [review]

File: config.json [hunk 2]
```diff
@@ -52,6 +52,11 @@
             "content": "https://github.com/cisco-lockhart/cdo-public-api-docs/blob/main/postman-collection.json",
             "type": "url"
           },
+          {
+            "title": "Postman Environment",
```
[the remark this answers is not in the document]
Comment (weilueluo) [reply config.json]: done 👍

File: postman-environment.json (hunk 1 above)
Comment (weilueluo) [reply postman-environment.json]: done 👍

**Pull request merged**